src/app/util/attribute-storage.cpp

```diff
@@ -1245,11 +1249,17 @@ void emAfLoadAttributeDefaults(EndpointId endpoint, bool ignoreStorage, Optional
 
                     if (ptr == nullptr)
                     {
+                        uint32_t defaultValueSizeForBigEndianNudger = 0;
+                        // Bypasses compiler warning about unused variable for little endian platforms.
+                        (void) defaultValueSizeForBigEndianNudger;
                         if ((am->mask & ATTRIBUTE_MASK_MIN_MAX) != 0U)
                         {
+                            // This is intentionally 2 and not DEFAULT_VALUE_SIZE_IN_DEFAULT_OR_MAX_MIN_STRUCT since min/max
+                            // attributes still are limited to size of 2-bytes.
                             if (emberAfAttributeSize(am) <= 2)
```

Comment:
Can we please do away with hard-coded constants and use `sizeof(EmberAfDefaultAttributeValue)` instead?

Reply:
You are right that there is a bug there that we actually should just always use 4 here since we have made the change to `defaultValue` in `EmberAfDefaultOrMinMaxAttributeValue` to `uint32_t`.

Making that change now